src/components/custom-aggrid/custom-aggrid-header.js

```diff
@@ -131,9 +131,10 @@ const CustomHeaderComponent = ({
                 }}
             >
                 <Autocomplete
-                    value={filterSelectedOption || ''}
+                    multiple
+                    value={filterSelectedOptions}
                     isOptionEqualToValue={(option, value) =>
-                        value === '' || option === value
+                        value === [] || option === value
```

Comment:
if value ===[] then it matches every options ?

Reply:
yes, it can be only an array. I also add [] as default value for filterSelectedOptions to avoid any problem

Reply:
yes, it can be only an array. I also add [] as default value for filterSelectedOptions to avoid any problem